articles/azure/azs-how-add-disks.md

```diff
@@ -0,0 +1,247 @@
+
+## Using the portal to add new disks to VM's
+
+1. In the portal choose **Virtual machines**.
+
+2. Select the VM you wish to add disks too.
+
+3. Navigate to **Disks** > **+Add data disk**.
+
+    ![Azure Stack disk creation 1](images/azs-browser-disk-creation.png)
+
+4. Edit the options for your new Disk.
+
+    - **Name**: This is what the disk will be called.
+
+    - **Source type**: Leave this as **New(empty disk)** as we are creating a new disk.
+
+    - **Account type**: This is where you choose between SSD and HDD.
+
+    - **Size**: This is where you specify the size of the new disk.
+
+    - **Storage container**: This is where you specify the storage container for the disk. Find the storage account where your VM's OS disk is located and then create a new.
+
+    - **Container**. Select this container as the location for the new disk.
+
+    - **Storage blob name**: This is the blob where the disk will be stored.
+
+    ![Azure Stack disk creation 2](images/azs-browser-disk-creation2.png)
+
+5. Select Ok and the new disks will be added to the VM.
+
+## Using the portal to add disks to existing VM's
+
+1. Ensure the existing disk you want to add to your VM is on you local machine as a .vhd file. Upload this .vhd to a new Container in your VM's storage account.
+
+2. After the .vhd file is uploaded, you are ready to attach the VHD to a VM. In the menu on the left, select **Virtual machines**.
+
+3. Chose the VM you wish to add the disk to.
+
+4. In **Source type** select **Existing blob**.
+
+5. In **Source blob** select **Browse**.
+
+6. Navigate to the storage account you uploaded the .vhd file to > select the container you uploaded the .vhd file to and click **Select**.
+
+    ![Azure Stack disk creation 3](images/azs-browser-disk-attaching.png)
+
+7. Finally click **OK** then **Save**.
+
+## Adding a disk to a VM that is already attached to another VM on Azure Stack
+
+Please ensure the disk is has been detached from its original VM.
+
+1. In the menu on the left, select **Virtual machines**.
+
+2. Chose the VM you wish to add the disk to.
+
+3. In **Source type** select **Existing blob**.
+
+4. In **Source blob** select **Browse**.
+
+5. Navigate to the storage container which contains the disk you wish to add to the VM, then navigate to the container where the disk resides.
+
+6. Finally click **Select** > **OK** > **Save**
+
+## Using PowerShell to add disks to a new VM
+
+In one of our other guides we showed you how to provision a VM (which can be found [here](azs-how-create-vm.md)) and the code in this guide will build off that. The Add-AzureRmVMDataDisk cmdlet adds a data disk to a virtual machine. You can add a data disk when you create a virtual machine, or you can add a data disk to an existing virtual machine. Specifying the VhdUri parameter to distribute the disks to different containers.  
+
+The following code will demonstrate how to add multiple disks to that VM on startup.
```

Comment:
The following code demonstrates how to add multiple disks to a VM on startup.

Reply:
done